recipes/omniscidb-cuda/recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,11 @@
+cuda_compiler:
```

Comment:
You can delete this file

Reply:
If I delete this file, the conda build fails:
```
conda.exceptions.ResolvePackageNotFound: 
  - cgo_linux-64
  - cuda_linux-64
```
What I am missing here?

Reply:
If you are building locally you have to do,
```
conda build recipe -m /path/to/conda_build_config.yaml
```
where `conda_build_config.yaml` is from https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml .

In a feedstock this `conda_build_config.yaml` file is reduced and saved in `.ci_support/linux*.yaml` files.

Reply:
Thanks, deleted.